apis/stash/v1beta1/types_helpers.go

```diff
@@ -0,0 +1,9 @@
+package v1beta1
+
+// TODO: complete
+func (t TargetRef) IsWorkload() bool {
+	if t.Kind == "Deployment" {
```

Comment:
What will happen for other workloads i.e. `StatefulSet` `DaemonSet`? However, it has been fixed in my workload controller PR.